qemu/tests/chardev_acpi.py

```diff
@@ -0,0 +1,35 @@
+from virttest import error_context
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    acpi description of serial and parallel ports incorrect
+    with -chardev/-device:
+    1) Boot guest A with isa-serial with tty chardev backend
+    2) Enter into guest A
+        udevadm info --query path --name /dev/ttyS0
+        !! --attribute-walk | grep looking
+    3) Boot guest B with -serial /dev/ttyS0
+    4) repeat step 2 for guest B
+    5) Check if the result are same for A and B.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    output = []
+    guest_cmd = params['guest_cmd']
+    for x in range(2):
+        if x >= 1:
+            params['serials'] = params['serials'].split()[:-1]
```

Comment:
Seems this line has some problem, can you check again? thanks
error log: AttributeError: 'list' object has no attribute 'split'

Reply:
Hi updated.
JOB ID     : c5d0b674a0b6b53a7f6083794b79782f172dee22
JOB LOG    : /root/avocado/job-results/job-2020-04-13T05.18-c5d0b67/job.log
 (1/1) Host_RHEL.m8.u1.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.chardev_acpi: PASS (79.41 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 81.65 s
JOB HTML   : /root/avocado/job-results/job-2020-04-13T05.18-c5d0b67/results.html

Reply:
Good catch